administrator/components/com_associations/views/association/view.html.php

```diff
@@ -62,6 +62,36 @@ public function display($tpl = null)
 			return false;
 		}
 
+		$this->app   = JFactory::getApplication();
+
+		$this->form  = $this->get('Form');
+		$input       = $this->app->input;
+
+		$this->referenceId = $input->get('id', 0, 'int');
+		$this->component   = AssociationsHelper::getComponentProperties($input->get('component', '', 'string'));
+
+		// Get reference language.
+		$this->table = clone $this->component->table;
+		$this->table->load($this->referenceId);
+
+		$this->associatedView = $this->component->item;
+		$extension            = $this->component->extension;
```

Comment:
where are this $this->associatedView and $extension used?
can be removed right?